README.md

```diff
@@ -29,23 +29,151 @@ frameworks (fltk, gtkmm, imgui, etc.), you can remove them with `git rm`:
 
 ## Dependencies
 
+Note about install commands: 
+- for Windows, we use [choco](https://chocolatey.org/install).
+- for MacOS, we use [brew](https://brew.sh/). 
+- In case of an error in cmake, make sure that the dependencies are on the PATH.
+
 ### Necessary Dependencies
-1. A C++ compiler that supports C++17. 
+1. A C++ compiler that supports C++17.
 See [cppreference.com](https://en.cppreference.com/w/cpp/compiler_support)
 to see which features are supported by each compiler.
 The following compilers should work:
-  * [gcc 7+](https://gcc.gnu.org/)
-  * [clang 6+](https://clang.llvm.org/)
-  * [Visual Studio 2017](https://visualstudio.microsoft.com/) or higher. 
-  The minimum compiler version is 19.15; this version ships with version 15.8 of the IDE. 
-2. [Conan](https://conan.io/) - it's recommended that you install using 
-[pip](https://pip.pypa.io/en/stable/) 
+
+	* [gcc 7+](https://gcc.gnu.org/)
+		<details>
+		<summary>Install command</summary>
+
+				- Debian/Ubuntu:
+					sudo apt install build-essential
+
+				- Windows:
+					choco install mingw -y
+
+				- MacOS:
+					brew install gcc
```

Comment:
Because these are used inside an HTML code, the Markdown syntax is lost. There is no way to make a list unless we use HTML code, which makes the readme cluttered for little benefit.

I am not sure about the benefit of adding prompt symbols. The cons are more than its single benefit. Powershell uses `>`, bash uses `$`, pip uses `>>>`. So there is no single prompt for everyone, which is confusing for beginners. Also, a symbol makes it harder to copy-paste.

Reply:
The markdown syntax is not lost; see example here:
https://github.com/ddalcino/cpp_starter_project/commit/5f5bc52bd00d8c25901eafcae11bcd2dd0d1eaec?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
This is fixed in https://github.com/lefticus/cpp_starter_project/pull/64/commits/cf5c33c45eb8bca5910a8649da4c3c9af0765273